### DISCUSSION TITLE
Camera Expo 51 (pausePreview)

### DISCUSSION THREAD

**Discussion by lukancosta** [expo/expo#29208] [Archived: Questions]
Any alternative for pausePreview() in the new Expo camera?

**Comment by vonkanehoffen** [comment]
Yeah I also really wanna know this @lukancosta 😄  There's this issue too: https://github.com/expo/expo/issues/26858

Best workaround I have so far is something like

```typescript
  const [picture, setPicture] = useState<CameraCapturedPicture | null>(null);
...
const picture = await cameraRef.current?.takePictureAsync();
setPicture(picture);
...
{picture ? (
        <Image source={{ uri: picture.uri }} style={StyleSheet.absoluteFill} />
      ) : (
        <CameraView
```
but that's really laggy and results in a flash of screen background while the Image elements loads the local uri

**Reply by IkeStudios** [reply]
I don't conditionally render CameraView, as unmounting and remounting that component causes the flickering. Instead, I only render the Image conditionally and set both the Image and CameraView components to have absolute positioning so they overlap (where the Image has a higher z-index). Instead of flickering, the CameraView seems to be normal for a split second before the Image loads in and superimposes it.

**Comment by egiesem** [comment]
any update?

**Reply by lukancosta** [reply]
Nothing so far

**Comment by matteomelotti** [comment]
I currently solved this importing the old `Camera as cameraRef` from `expo-camera/legacy` and using `pausePreview`/`resumePreview` as before

**Comment by tsapeta** [comment]
Pausing and resuming the preview in the new Camera module will be available in SDK 52. The PR was already merged: https://github.com/expo/expo/pull/30666
